source/use-cases/local-master-key.txt

```diff
@@ -1,89 +1,23 @@
 A. Create a Local Master Key
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Client-Side Field Level Encryption for MongoDB uses the envelope encryption mechanism in which there are two types of keys: 1) **data encryption keys** which are used to encrypt/decrypt the sensitive data fields and 2) a **master key** that is used to encrypt the **data encryption keys**.
+Client-Side Field Level Encryption for MongoDB uses an encryption strategy called envelope encryption in which keys used to encrypt/decrypt data (called **data encryption keys**) are encrypted with another key (called the **master key**). 
 
-The **master key** is stored separately in a Key Management System (KMS) in order to prevent the **data encryption keys** from being used directly to access the sensitive data.
+This strategy bolsters the security, minimizes encryption/decryption overhead, and allows the data encryption keys to be stored in the same replica set as the encrypted data. If a basic encryption strategy was used instead of envelope encryption, the computational and network load of encryption and decryption would need to be handled by the system that manages your keys. For more information on envelope encryption and key management concepts, read `this AWS article <https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#enveloping>`_.
 
-The **data encryption keys** can safely be stored in a MongoDB collection alongside the encrypted sensitive data fields since they have been encrypted by the **master key**.
+The master key is stored separately in a Key Management System (KMS) in order to prevent the data encryption keys from being used directly to access the sensitive data.
 
-In Client-Side Field Level Encryption, a client that has access to both **master key** and **data encryption keys** is necessary to perform encryption/decryption.
+The data encryption keys can safely be stored in a MongoDB collection alongside the encrypted sensitive data fields since they have been encrypted by the master key.
```

Comment:
These two sentences both seem out of place - we just finished up a paragraph about envelope encryption and referred users to an outside guide, then these carry on talking about envelope encryption concepts.

Can we make these more contextually relevant to MongoDB FLE specifically rather than envelope encryption?
- Should these move to just after the opening sentence (i.e. above the previous paragraph)?
- Should we combine these with the previous paragraph? "alongside the encrypted data" seems like an area that envelope encryption gets a big win vs basic.

Also (nit) they both use passive voice but should preferably be active.

Reply:
I agree. I think it needs a lead-in and also should probably only include info about the master key since that's all this step is concerned with.